### ISSUE TITLE
USWDS - Table: Change caption text size to match body and table text size

### ISSUE THREAD

**Issue by amyleadem** [uswds/uswds#6253]
## Summary
Can you please fix the style of default font size for `<caption>` code within [tables](https://designsystem.digital.gov/components/table/)  so that they’re not smaller than all text in the table and on the page?

Here's an example of a page with this issue so you can see what we're talking about: https://digital.gov/about/manage-content/

> [!note]
> This request originated in this [Slack thread](https://gsa-tts.slack.com/archives/C050HRGN7/p1734095567505819).

**Comment by bonnieAcameron** [comment]
### Background/user need: 

The Digital.gov team are trying to make our tables more accessible and more contextual by ensuring that they have titles. 

In the example on [one of our about pages](https://digital.gov/about/manage-content/), you can see that we've used the `<caption>` to _look_ like the title, but it doesn't _function_ like the title. 

### Alternatives we've considered

After discussing with Alex H., we could potentially update the `<caption>` text to say something like: 

`<caption> Table 1: A table that tells you about stuff </caption>`

### Potential solutions, not exhaustive

1. Update the guidance for table <captions> to clarify their usage
2. Update the guidance and/or code for table title fields
3. Update the styling of the <captions> to be larger in text if it is to appear above the table and/or co-function as a title

**Comment by dct2023** [comment]
Adding this MDN table example from James for reference:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption#result

**Comment by dct2023** [comment]
@amycole501 to provide additional details on what we might need to make these changes.

**Comment by amycole501** [comment]
Draft [research and guidance change](https://docs.google.com/document/d/183yNq2Pf-dASlI89ijCVPWyB70FC5ct6AikXtlijnls/edit?usp=sharing).

**Comment by amycole501** [comment]
Based on my research I agree that we should update our caption element style to appear larger and more in keeping with a title, which is the function the `caption` element serves.
